src/Microsoft.SqlTools.ServiceLayer/LanguageServices/Contracts/Completion.cs

```diff
@@ -23,6 +24,13 @@ public static readonly
             RequestType<CompletionItem, CompletionItem>.Create("completionItem/resolve");
     }
 
+    public class CompletionExtLoadRequest
```

Comment:
>CompletionExtLoadRequest [](start = 17, length = 24)

Also please update [https://github.com/microsoft/sqltoolsservice/blob/master/docs/guide/jsonrpc_protocol.md](https://github.com/microsoft/sqltoolsservice/blob/master/docs/guide/jsonrpc_protocol.md) with the new request/response items. I'd suggest making a new section called "Language Service Protocol Extensions" since this isn't part of the official language service protocol.

Reply:
Updated the doc.